clio-client/src/test/scala/org/broadinstitute/client/commands/MoveWgsUbamSpec.scala

```diff
@@ -105,6 +105,36 @@ class MoveWgsUbamSpec extends BaseClientSpec {
     }
   }
 
+  it should "throw and exception if given a non-GCP unmapped bam" in {
+    a[Exception] should be thrownBy {
+      val config = BaseArgs(
+        command = Some(MoveWgsUbam),
+        flowcell = testFlowcell,
+        lane = testLane,
+        libraryName = testLibName,
+        location = Some("OnPrem"),
+        bearerToken = testBearer,
+        ubamPath = testUbamCloudDestinationPath
+      )
+      succeedingDispatcher.dispatch(config)
+    }
+  }
+
+  it should "throw and exception if the destination path is not in GCP" in {
+    a[Exception] should be thrownBy {
+      val config = BaseArgs(
+        command = Some(MoveWgsUbam),
+        flowcell = testFlowcell,
+        lane = testLane,
+        libraryName = testLibName,
+        location = Some("OnPrem"),
```

Comment:
This location should be `Some("GCP")` to really test it's the `ubamPath` raising the error.

Reply:
Damn copy pasta.